docs/Development-Guides/Field-Support/Benefits-Support.md

```diff
@@ -0,0 +1,786 @@
+**FSA Medical** | <strong><span style="color:red">x</span></strong> | <strong><span style="color:green">✓</span></strong> | <strong><span style="color:red">x</span></strong> | <strong><span style="color:green">✓</span></strong> |
+**Section 125 Dental** | <strong><span style="color:red">x</span></strong> | <strong><span style="color:green">✓</span></strong> | <strong><span style="color:red">x</span></strong> | <strong><span style="color:green">✓</span></strong> |
+**Section 125 Medical** | <strong><span style="color:red">x</span></strong> | <strong><span style="color:green">✓</span></strong> | <strong><span style="color:red">x</span></strong> | <strong><span style="color:green">✓</span></strong> |
+**Section 125 Vision** | <strong><span style="color:red">x</span></strong> | <strong><span style="color:green">✓</span></strong> | <strong><span style="color:red">x</span></strong> | <strong><span style="color:green">✓</span></strong> |
+**Commuter (pre-tax)** | <strong><span style="color:red">x</span></strong> | <strong><span style="color:green">✓</span></strong> | <strong><span style="color:red">x</span></strong> | <strong><span style="color:green">✓</span></strong> |
+**Custom pre-tax** | n/a | n/a | n/a | n/a |
+**Custom post-tax** | <strong><span style="color:red">x</span></strong> | <strong><span style="color:green">✓</span></strong> | <strong><span style="color:red">x</span></strong> | <strong><span style="color:green">✓</span></strong> |
+
+<!--
+type: tab
+title: Supported Features
+-->
+
+Benefit | Features
+--------|----------
+**401(k)** | `company_contribution`: `percent`<br>`employee_deduction`: `fixed`, `percent`<br>`catch_up`
+**Roth 401(k)** | `company_contribution`: `percent`<br>`employee_deduction`: `fixed`, `percent`<br>`catch_up`
+**401(k) Loan** | `employee_deduction`: `fixed`
+**403(b)** | `company_contribution`: `percent`<br>`employee_deduction`: `fixed`, `percent`
+**Roth 403(b)**| `company_contribution`: `percent`<br>`employee_deduction`: `fixed`, `percent`
+**457**| `employee_deduction`: `fixed`, `percent`<br>`catch_up`
+**Roth 457** | `employee_deduction`: `fixed`, `percent`<br>`catch_up`
+**Simple IRA**| `company_contribution`: `fixed`, `percent`<br>`employee_deduction`: `fixed`, `percent`<br>`catch_up`
+**HSA (pre-tax)**| `employee_deduction`: `fixed`, `percent`<br>`hsa_contribution_limit`: `individual`, `family`
+**HSA (post-tax)**| `company_contribution`: `fixed`, `percent`<br>`employee_deduction`: `fixed`, `percent`<br>`hsa_contribution_limit`: `individual`, `family`
+**FSA Dependent** | `employee_deduction`: `fixed`
+**FSA Medical** | `employee_deduction`: `fixed`
+**Section 125 Dental** | `employee_deduction`: `fixed`
+**Section 125 Medical**| `employee_deduction`: `fixed`
+**Section 125 Vision** | `employee_deduction`: `fixed`
+**Commuter (pre-tax)** | `employee_deduction`: `fixed`
+**Custom post-tax** | `employee_deduction`: `fixed`, `percent`
```

Comment:
I don't know how much of this is appropriate to be nitpicky about. I think the "company contribution", "employee deduction", and "catch up" fields could be clearer as their own columns.

The below is a bit "noisy". 

<img width="492" alt="Screen Shot 2023-04-27 at 12 57 39 PM" src="https://user-images.githubusercontent.com/2038511/234977371-0a97f87e-62bc-47b1-970f-8c3a867e1a01.png">